src/main/java/models/Game.java

```diff
@@ -35,6 +41,41 @@ public void buildDeck()
 		}
 	}
 
+	public void addCustomCardToUser()
+	{
+		Card addMe = new Card();
+		addMe.setCard(Suit.Spades,2);
+		user.hand.add(addMe);
+	}
+
+	public void addCustomCardToDealer()
+	{
+		Card addMe = new Card();
+		addMe.setCard(Suit.Spades,3); //different to test who wins
+		dealer.hand.add(addMe);
+	}
+
+
+	public void dealCardToUser()
+	{
+		user.hand.add(dealCard());
+	}
+
+	public void dealCardToDealer()
+	{
+		dealer.hand.add(dealCard());
+	}
+
+	public void whoWon()
+	{
+		int dealerScore = dealer.calcScore(dealer.hand);
+		int userScore = user.calcScore(user.hand);
+		if (dealerScore > userScore)
```

Comment:
Check out new code, did basically what you said minus adding the score set to zero.